Patch from upstream CVS, generated on 20100504 via

  cvs -d:pserver:anonymous@cvs.savannah.nongnu.org:/sources/spamass-milt co spamass-milt
  cd spamass-milt && cvs diff -r RELEASE_0_3_1


Index: README
===================================================================
RCS file: /sources/spamass-milt/spamass-milt/README,v
retrieving revision 1.6
retrieving revision 1.7
diff -u -p -r1.6 -r1.7
--- README	15 Aug 2005 15:10:47 -0000	1.6
+++ README	19 Apr 2006 15:40:36 -0000	1.7
@@ -1,4 +1,4 @@
-$Id: spamass-milter.20100504.patch,v 1.1 2010/06/04 21:23:02 gdt Exp $
+$Id: spamass-milter.20100504.patch,v 1.1 2010/06/04 21:23:02 gdt Exp $
 
 Spamass-Milter, a small
  - SpamAssassin Sendmail Mail Filter (Milter) Plugin 
@@ -57,6 +57,7 @@ adding the lines
 INPUT_MAIL_FILTER(`spamassassin', `S=local:/var/run/sendmail/spamass.sock, F=, T=C:15m;S:4m;R:4m;E:10m')dnl
 define(`confMILTER_MACROS_CONNECT',`t, b, j, _, {daemon_name}, {if_name}, {if_addr}')dnl
 define(`confMILTER_MACROS_HELO',`s, {tls_version}, {cipher}, {cipher_bits}, {cert_subject}, {cert_issuer}')dnl
+define(`confMILTER_MACROS_ENVRCPT',`r, v, Z')dnl
 
 should do the trick. Of course you need to modify the path of the
 socket if you put another one into the startup script. The timeouts
Index: spamass-milter.cpp
===================================================================
RCS file: /sources/spamass-milt/spamass-milt/spamass-milter.cpp,v
retrieving revision 1.90
retrieving revision 1.91
diff -u -p -r1.90 -r1.91
--- spamass-milter.cpp	23 Mar 2006 21:41:36 -0000	1.90
+++ spamass-milter.cpp	24 Jul 2006 19:59:17 -0000	1.91
@@ -1,6 +1,6 @@
 // 
 //
-//  $Id: spamass-milter.20100504.patch,v 1.1 2010/06/04 21:23:02 gdt Exp $
+//  $Id: spamass-milter.20100504.patch,v 1.1 2010/06/04 21:23:02 gdt Exp $
 //
 //  SpamAss-Milter 
 //    - a rather trivial SpamAssassin Sendmail Milter plugin
@@ -127,7 +127,7 @@ int daemon(int nochdir, int noclose);
 
 // }}} 
 
-static const char Id[] = "$Id: spamass-milter.20100504.patch,v 1.1 2010/06/04 21:23:02 gdt Exp $";
+static const char Id[] = "$Id: spamass-milter.20100504.patch,v 1.1 2010/06/04 21:23:02 gdt Exp $";
 
 struct smfiDesc smfilter =
   {
@@ -1002,9 +1002,9 @@ mlfi_envrcpt(SMFICTX* ctx, char** envrcp
 
 		assassin->output((string)
 			"Received: from "+macro_s+" ("+macro__+")\r\n\t"+
-			"by "+macro_j+"("+macro_v+"/"+macro_Z+") with "+macro_r+" id "+macro_i+"\r\n\t"+
+			"by "+macro_j+"("+macro_v+"/"+macro_Z+") with "+macro_r+" id "+macro_i+";\r\n\t"+
 			macro_b+"\r\n\t"+
-			"(envelope-from "+assassin->from()+"\r\n");
+			"(envelope-from "+assassin->from()+")\r\n");
 
 	} else
 		assassin->output((string)"X-Envelope-To: "+envrcpt[0]+"\r\n");
